content/javascript/concepts/data-types/data-types.md

```diff
@@ -14,20 +14,23 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Data types are how data in programming is classified. In JavaScript, there are eight fundamental data types:
-
-| Data Type   | Description                                                                                                                                   |
-| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Number`    | Any number, including numbers with decimals: `1`, `-2`, `99`, `3.14`.                                                                         |
-| `BigInt`    | Any number, greater than 2^53-1 or less than -(2^53-1) with `n` appended to the number: `1234567890123456n`.                                  |
-| `String`    | Any grouping of characters on your keyboard (letters, numbers, spaces, symbols, etc.) surrounded by single quotes `''` or double quotes `""`. |
-| `Boolean`   | This data type only has two possible values — either `true` or `false`.                                                                       |
-| `Null`      | This data type represents the intentional absence of a value, and is represented by the keyword `null`.                                       |
-| `Undefined` | This data type is denoted by the keyword `undefined`. It also represents the absence of a value though it has a different use than `null`.    |
-| `Symbol`    | A newer feature to the language, symbols are unique identifiers, useful in more complex coding. No need to worry about these for now.         |
-| `Object`    | Collections of related data.                                                                                                                  |
-
-The first 7 of those types are considered primitive data types. They are the most basic data types in the language. Objects are a little more complex.
+**Data types** are how data in programming is classified. In JavaScript, there are eight fundamental data types:
+## Primitive Data Types
+The below 7 in the table are considered primitive data types. They are the most basic data types in the language. Objects are a little more complex.
+
+| Data Type | Wrapper Object   | Description                                                                                                                                   |
+|-----| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `number` | [`Number`](../number-methods/number-methods.md)    | Any number, including numbers with decimals: `1`, `-2`, `99`, `3.14`.                                                                         |
```

Comment:
Make sure to link out properly, the urls should look like https://www.codecademy.com/resources/docs/...